modules/admission-webhook-types.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-types_{context}"]
+= Types of admission webhooks
+
+Cluster administrators can include _mutating admission webhooks_ or _validating admission webhooks_
+in the admission chain of the API server.
+
+[id="mutating-admission-webhooks_{context}"]
+*Mutating admission webhooks* are invoked during the mutation phase of the admission process, which allows modification of the resource content before it is persisted. One example of a mutating admission webhook is the Pod Node Selector Pod Node Selector feature, which uses an annotation on a namespace to find a label selector and add it to the pod specification.
+
+[id="mutating-admission-webhooks-sample"]
```

Comment:
The anchor needs _{context} added. If this module were to be included more than once in the doc set, you'd run into a duplicate ID error without the _{context}.

```
 [id="mutating-admission-webhooks-sample_{context}"]
```

Reply:
Resolved.